Vision/Vision.bs

```diff
@@ -106,7 +106,7 @@ Markup Shorthands: markdown yes
 	where diverse voices from around the world
 	and from different organizations and industries
 	work together to evolve the web by building consensus
-	on voluntary global standards for Web technologies.
+	on global standards, some voluntary and some not, for Web technologies.
```

Comment:
```suggestion
	on global standards for Web technologies.
```

might be simpler.

Reply:
I agree, if voluntary is controversial, better to just not say it. Otherwise the PR as proposed opens a question in readers' minds about what's voluntary and what's not.

Reply:
I completely share your concerns and debated it with myself (and so I'm glad to do so with others — they're usually more interesting :). I think that there are two defensible positions:

1. It's controversial, and therefore we shouldn't mention it.
2. It's controversial, and therefore we should mention it.

My initial instinct was to go with (1), but then I wondered what we should expect from this document and changed my mind. Do we want this to be a document that helps us have hard conversations or one that pretends the hard conversations don't exist?

With respect to the question of voluntary standards, we should constantly be asking ourselves questions like:

* Is this something that should be voluntary or not?
* Are we respecting the Priority in how we deploy enforcement?
* Is this a cautious and responsible use of the power we have?

We demonstrate leadership when we face these questions; we fail our community when we pretend these questions aren't to be asked or don't correspond to the experienced reality of millions of sites and billions of people. I believe that this document would be stronger if it did more of the former.

I don't disagree that it makes the sentence clunky. One alternative would be to apply @chaals's change above and then include an additional bullet point under Operational Principles. Something like (wordsmithing welcome):

* **Prudent Enforcement**. We take great care to deploy the coercive power of standards thoughtfully, responsibly, and in accordance with clearly established principles.

I don't believe that the reality is on par with this principle, though several of us are trying, but we should be aspirational!

Reply:
I'll follow up in #177...